src/app.ts

```diff
@@ -63,6 +65,30 @@ const authenticatedRouter = new Router()
         ctx.state.toMember = midQuery.docs[0];
         return await next();
     })
+    .post('/api/members/:mid/request_invite', async ctx => {
+        const authUid = ctx.state.user.uid;
+        const authMember = await members.doc(authUid).get();
+        try {
+            const newOperation = {
+                creator_mid: authMember.get('mid'),
+                creator_uid: authUid,
+                op_code: 'REQUEST_INVITE',
+                data: {
+                    full_name: ctx.request.body.fullName,
+                    to_mid: ctx.state.toMember.get('mid'),
```

Comment:
For the centralized operations, no need to store the redundant to_mid (though we can choose to transform it when put in blockchain.

Reply:
agreed

Reply:
but see above